Packages/vcs/Lib/vcsvtk/fillareautils.py

```diff
@@ -33,8 +34,14 @@ def make_patterned_polydata(inputContours, fillareastyle=None,
     # Scaled the size to 2 times to make the pattern image of a finer resolution
     xBounds = bounds[1] - bounds[0]
     yBounds = bounds[3] - bounds[2]
-    xres = int(4.0*xBounds)
-    yres = int(4.0*yBounds)
+    if xBounds <= 1 and yBounds <= 1 and size is not None:
+        xBounds *= size[0]
```

Comment:
Here's the reason for this pull request:

```python
import vcs
fa = vcs.createfillarea()
fa.x = [0, 0, 1, 1]
fa.y = [0, 1, 1, 0]
fa.style = "hatch"
fa.color = 242
fa.index = 5
canvas = vcs.init()
canvas.plot(fa)
```

Using master, you get this:
![busted](https://cloud.githubusercontent.com/assets/718512/10888204/bbbfb298-813f-11e5-9224-310eef5ea529.png)

Using this branch, you get this:
![size_adjust](https://cloud.githubusercontent.com/assets/718512/10888205/bbc0061c-813f-11e5-94e1-01b2c02a34cb.png)

For fillareas, the bounds are always between 0 and 1; this means we wind up applying a 16x32 pixel pattern to a *giant* fillarea. If the texture were being applied in a tiling manner, this wouldn't be an issue, but since we create one texture map for everything, this is going to look really terrible.

Also, no, I don't think it will create different results for different displays. The size passed in is from `vtkRenderWindow.GetSize()`, so I think it should always generate the same output for a given render window size.

Also, I tested with multiple resolutions and it appears to have always been the same. That said, I should make it so the test specifies a window size.